Controllers/CHumidityController.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * CHumidityController.h
+ *
+ *  Created on: Aug. 11, 2022
+ *      Author: samk
+ */
+
+#ifndef CHUMIDITYCONTROLLER_H_
+#define CHUMIDITYCONTROLLER_H_
+
+#include "CBME280.h"
+#include "CController.h"
+#include "CPIDLoop.h"
+#include "IHardwareMap.h"
+
+#define CHANNEL_NUMBER 8
```

Comment:
Why do we have multiple channels?

Reply:
In the temperature controller code, it looks like a heater is assigned per channel. In my humidity controller code, I utilize all the heaters in changing the humidity.